components/IconGlobe.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <svg width="46" height="47" viewBox="0 0 46 47" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
`width="46" height="47"` if you use like this the component that's mean that the dimension is fixed.

To allow to choose the dimension from the parent you need to remove that part.
So will be the parent with the tailwindcss class to choose the dimensions